src/nat/builder/child_builder.py

```
@@ -336,3 +346,30 @@ async def get_middleware(self, middleware_name: str | MiddlewareRef) -> Middlewa
     def get_middleware_config(self, middleware_name: str | MiddlewareRef) -> MiddlewareBaseConfig:
         """Get the configuration for middleware."""
         return self._workflow_builder.get_middleware_config(middleware_name)
+
+    @staticmethod
+    @contextmanager
+    def use(config: TypedBaseModel, builder: Builder) -> Generator["ChildBuilder", None, None]:
```

Ruff (0.14.10)

Unused static method argument: `config`

(ARG004)